Formula/mycli.rb

```diff
@@ -20,6 +20,7 @@ class Mycli < Formula
   end
 
   depends_on "rust" => :build
+  depends_on "cffi"
```

Comment:
Shouldn't we also add `depends_on "pycparser"`?

Reply:
yeah, I think so

Reply:
the formula does not depend on `pycparser`?

Reply:
Sorry if I don't understand -- the `cffi` and `pycparser` resources were removed and only `depends_on "cffi"` was added. It doesn't seem like the formula already depends on `pycparser`. Should it be added or am I missing something here?

Reply:
Ah okay, just realised `cffi` depends on `pycparser` already.

Reply:
I think the setup.py for mycli does not specify `pycparser` is actually the thing in here.